docs/concepts/evaluations/index.md

```diff
@@ -0,0 +1,3 @@
+# Evaluations
```

Comment:
see commend on structure, I think this should just be merged with the other pages into a single concepts page (at least for now). Each of these docs can be a high level section.